src/util/path.ts

```diff
@@ -935,7 +936,17 @@ export const getRegularPolygonPath = (numVertexes, radius) => {
 /**
  * Join path commands to go back to svg format
  * @param {Array} pathData fabricJS parsed path commands
+ * @param {number} fractionDigits  number of fraction digits to "leave"
  * @return {String} joined path 'M 0 0 L 20 30'
  */
-export const joinPath = (pathData) =>
-  pathData.map((segment) => segment.join(' ')).join(' ');
+export const joinPath = (pathData: PathData, fractionDigits: number) =>
+  pathData
+    .map((segment) => {
+      return segment
+        .map((arg, i) => {
+          if (i === 0) return arg;
+          return toFixed(arg, fractionDigits);
```

Comment:
fractionDigits should go back to optional and

```suggestion
          return isNumber(fractionDigits) ? toFixed(arg, fractionDigits) : arg;
```

Or something similar to this.

Reply:
That is what I thought but then again looking in a broader context it is an svg util and everything in svg output forces the config value without giving a way out

Reply:
Made `fractionDigits` optional.

Reply:
i used `joinPath` a bunch of time on my own to do things in json export for example or manual things. I use it outside the context of svg export.